### DISCUSSION TITLE
How to fix 99% of your issues. [CUDA]

### DISCUSSION THREAD

**Discussion by Ku-Tadao** [hacksider/Deep-Live-Cam#242] [General]
Simple, just follow the guide and install 

the most important thing isssss, install Python 3.10.X
https://www.python.org/ftp/python/3.10.11/python-3.10.11-amd64.exe

after that, simply follow the guide starting from step 1, and i can promise you that it suddenly becomes very easy without that many bugs.

if you still have issues and you're on Python 3.1.11 (that version specifically), then i'm willing to dig into it for people aswell.

**Comment by maykl19825-pixel** [comment]
The easiest way is instal anaconda)
